### DISCUSSION TITLE
建议增加给软件加入计划/标签功能，当软件运行时，自动统计入计划/标签的运行时间

### DISCUSSION THREAD

**Discussion by luxaug** [shion-app/shion#49] [Ideas]
举例而言，设定计划：学习，学习下设置标签：写作，给word软件设置学习/写作标签，则word运行时间自动统计进学习/写作的专注时间

**Comment by hanaTsuk1** [comment]
不考虑，具体原因见[不完美的自动化](https://shion.app/zh/chat/imperfect-automation)

太长不想看的话，简单来说，做起来有误差，麻烦，不如手动

**Reply by luxaug** [reply]
好的，了解了，谢谢
